korebuild-lock.txt

```diff
@@ -1,2 +1,2 @@
-version:2.1.0-preview1-1008
-commithash:e82dffbe8cf67ad9c07b4385cb6a1a1fc8be9fb1
+version:2.1.0-preview1-1010
```

Comment:
We probably want to keep dev using preview2 build tools. Latest is 2.1.0-preview2-15692